ultimarc/devices/_base.py

```diff
@@ -33,15 +35,15 @@
     'd209:1602': 'Aimtrak Lightgun #2',
     'd209:1603': 'Aimtrak Lightgun #3',
     'd209:1604': 'Aimtrak Lightgun #4',
+    'd209:0440': 'Mini-PAC #1',
 }
```

Comment:
Is it possible to set a new Mini-PAC id like the lightgun?  So you can have a MP with a vendor/product id of 'd209:0441'?  If so how many different ids can you set?

Reply:
Yes there are more id available.  Looks like from the spreadsheet it can be up to 8 different devices.  This will apply across all the devices.  This list will get pretty long.  Once I start filling out the details of what the python device can do I will add the additional devices in.

Reply:
Sounds good.